helm-charts/stable/amazon-eks-pod-identity-webhook/templates/job-Role.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2020 Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: pod-identity-webhook-job
+  namespace: default
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - serviceaccounts
+  - secrets
+  verbs:
+  - get
```

Comment:
Why does this need to get all secrets in the namespace?

Reply:
It needs whatever is defined in the script below, so not sure if we can specify a specific secret to constrain on?
```
secret_name=$(kubectl get sa default -o jsonpath='{.secrets[0].name}')
CA_BUNDLE=$(kubectl get secret/$secret_name -o jsonpath='{.data.ca\.crt}' | tr -d '\n')
```

Reply:
Ah in that case it shouldn't need secret access

```
CA_BUNDLE=$(cat /var/run//secrets/kubernetes.io/serviceaccount/ca.crt)
```